packages/truffle-core/lib/commands/migrate.js

```diff
@@ -135,25 +155,25 @@ var command = {
           }
         });
       }
-    };
+    }
 
-    async function executePostDryRunMigration(buildDir){
+    async function executePostDryRunMigration(buildDir) {
       let accept = true;
 
-      if (options.interactive){
+      if (options.interactive) {
         accept = await Migrate.acceptDryRun();
       }
 
-      if (accept){
-        const environment = require('../environment');
-        const NewConfig = require('truffle-config');
+      if (accept) {
+        const environment = require("../environment");
+        const NewConfig = require("truffle-config");
         const config = NewConfig.detect(options);
 
         config.contracts_build_directory = buildDir;
         config.artifactor = new Artifactor(buildDir);
         config.resolver = new Resolver(config);
 
-        environment.detect(config, function(err) {
+        environment.detect(config, () => {
```

Comment:
(We probably shouldn't ignore errors from doing `Environment.detect()`... unless these get thrown)